src/lib/utils/options/sources/typescript.ts

```diff
@@ -72,15 +72,6 @@ export class TypeScriptSource extends OptionsComponent {
                 }
         }
 
-        switch (option.paramType) {
```

Comment:
No longer necessary since this is only used when printing help output, and TypeDoc hasn't printed TypeScript options in help since #569 was resolved months ago.